plugins/outputs/http/README.md

```diff
@@ -0,0 +1,36 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`
+
+It requires a `url` name.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.graylog]]
+  ## Will be transmitted telegraf metrics to the HTTP Server using the below URL.
+  url = "http://127.0.0.1:8080/metric"
+  ## HTTP Content-Type. Default : application/json
+  content_type = "application/json"
+  ## Set the number of times to retry when the status code is not 200 or an error occurs during HTTP call. Default: 3
+  retry = 3
```

Comment:
Oh, Really? It sounds nice. 
Is that functionality still available? 
If so, where can I see that part?